gslib/utils/posix_util.py

```diff
@@ -430,11 +430,11 @@ def ParseAndSetPOSIXAttributes(path, obj_metadata, is_rsync=False,
     if atime > NA_TIME and mtime > NA_TIME:
       # Set both atime and mtime.
       os.utime(path, (atime, mtime))
-    elif atime > NA_TIME and mtime <= NA_TIME:
+    elif atime > NA_TIME >= mtime:
```

Comment:
I think this actually makes the predicate harder to understand - we should be wary of adding the three-way comparisons unless they're semantically checking "make sure that <<value in the middle>> is between the two values surrounding it".

Reply:
It can be reverted, but I made this change because it seemed to me the more readable way of displaying that information. Maybe that's just me though.

Reply:
This is probably fine --  looking at it again, using lt/gt operators to compare against an "NA" value is confusing enough in the first place. We should really go in an add helper methods or figure out a way to make this read a bit more cleanly, but it's fine for now.